great_expectations/profile/columns_exist.py

```diff
@@ -1,20 +1,21 @@
 from .base import DatasetProfiler
 
 import warnings
-from six import string_types
 
 from ..dataset.util import create_multiple_expectations
 
 
 class ColumnsExistProfiler(DatasetProfiler):
 
     @classmethod
-    def _profile(cls, dataset):
+    def _profile(cls, dataset, configuration=None):
         """
         This function will take a dataset and add expectations that each column present exists.
 
         Args:
             dataset (great_expectations.dataset): The dataset to profile and to which to add expectations.
+            :type configuration: dict
+            :param configuration: Configuration for select profilers.
```

Comment:
Does our sphinx builder handle this style of docstring syntax?

Reply:
fixed